src/identifiers.md

```diff
@@ -5,10 +5,12 @@
 > &nbsp;&nbsp; &nbsp;&nbsp; [`a`-`z` `A`-`Z`]&nbsp;[`a`-`z` `A`-`Z` `0`-`9` `_`]<sup>\*</sup>\
 > &nbsp;&nbsp; | `_` [`a`-`z` `A`-`Z` `0`-`9` `_`]<sup>+</sup>
 >
+> RAW_IDENTIFIER : `r#` IDENTIFIER_OR_KEYWORD
+>
 > IDENTIFIER :\
-> IDENTIFIER_OR_KEYWORD <sub>*Except a [strict] or [reserved] keyword*</sub>
+> IDENTIFIER_OR_KEYWORD <sub>*Except a [strict] or [reserved] keyword*</sub> | RAW_IDENTIFIER
 
-An identifier is any nonempty ASCII string of the following form:
+A normal (non-raw) identifier is any nonempty ASCII string of the following form:
```

Comment:
I don't think this line needs to change. I feel like it draws attention away from basic identifiers in a way that could be confusing.

Reply:
Yeah, I thought about this... you definitely have a point. I just wasn't sure if it would be misleading to "go back" on the initial definition of an identifier slightly. But fair enough.